kuma/users/checks.py

```diff
@@ -31,11 +30,16 @@ def stripe_check(app_configs, **kwargs):
                     id=STRIPE_PLAN_INACTIVE_ERROR,
                 )
             )
+    except stripe.error.StripeError as error:
+        errors.append(
+            Error(f"unable to retrieve stripe plan: {error}", id=STRIPE_PLAN_ERROR)
+        )
 
+    try:
         create_missing_stripe_webhook()
     except stripe.error.StripeError as error:
         errors.append(
-            Error(f"unable to retrieve stripe plan: {error}", id=STRIPE_PLAN_ERROR)
+            Error(f"unable get or create strip webhooks: {error}", id=STRIPE_PLAN_ERROR)
```

Comment:
Technically, the function should be renamed to `get_or_create_missing_stripe_webhook` or something better because it's not 100% creating. But one thing at a time. At least now this error message is slightly more accurate.